tests/test_cli/test_run.py

```diff
@@ -31,7 +31,7 @@
 
 import pytest
 import yaml
-from aea_ledger_fetchai import FetchAICrypto
+from aea_ledger_ethereum import EthereumCrypto
```

Comment:
`Error: Specified default ledger 'ethereum' not found in available addresses of types: [fetchai]`